kura/test/org.eclipse.kura.container.provider.test/src/test/java/org/eclipse/kura/container/provider/ContainerInstanceTest.java

```diff
@@ -57,6 +62,7 @@ public class ContainerInstanceTest {
     private static final String CONTAINER_VERIFY_TLOG = "container.signature.verify.transparency.log";
     private static final String CONTAINER_REGISTRY_USERNAME = "registry.username";
     private static final String CONTAINER_REGISTRY_PASSWORD = "registry.password";
+    private static final String CONTAINER_ENFORCEMENT_DIGEST = "enforcement.digest";
```

Comment:
I noticed this only now but... for consistency, since it's releated to the other parameters, would it make sense to call it:

```suggestion
    private static final String CONTAINER_ENFORCEMENT_DIGEST = "container.signature.enforcement.digest";
```

Obviously such rename would require a separate PR. I'm just throwing this here in the meantime...

Reply:
Done in #5208